ui/css/perf.css

```diff
@@ -459,6 +459,17 @@ li.pagination-active.active > button {
   color: black !important;
 }
 
+.page-item.disabled .page-link {
+  color: lightgray !important;
+}
```

Comment:
We could also add the `cursor: not-allowed;` or `cursor: default;` and not pointer. This way it suggests that the buttons are not clickable.